src/core/Flyout.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import {
+  NavigationMenu,
+  NavigationMenuItem,
+  NavigationMenuList,
+  NavigationMenuTrigger,
+  NavigationMenuContent,
+  NavigationMenuViewport,
+  NavigationMenuLink,
+} from "@radix-ui/react-navigation-menu";
+import cn from "./utils/cn";
+
+const NavigationMenuDemo = ({
+  menuItems,
+  navMenuStyling,
+  flyOutStyling,
+}: {
+  menuItems: {
+    label: string;
+    content?: React.ReactNode;
+    link?: string;
+    panelStyling?: string;
+  }[];
+  navMenuStyling?: string;
+  flyOutStyling?: string;
+}) => {
+  const menuLinkStyles =
+    "ui-text-menu3 text-neutral-1000 font-bold px-12 py-8 hover:bg-neutral-100 hover:text-neutral-1300 cursor-pointer flex items-center";
+  const viewPortStyling =
+    "relative overflow-hidden w-full h-[var(--radix-navigation-menu-viewport-height)] origin-[top_center] transition-[width,_height] duration-300 data-[state=closed]:animate-scaleOut data-[state=open]:animate-scaleIn sm:w-[var(--radix-navigation-menu-viewport-width)]";
+  const boxShadowViewport =
```

Comment:
Can we not use one of the existing defined shadows for this? https://ably.github.io/ably-ui/?path=/docs/styles-shadows--docs They may not be spot on but I would suggest picking the closest one

Reply:
ahhh that explains this label in figma.
<img width="190" alt="Screenshot 2025-01-31 at 14 46 22" src="https://github.com/user-attachments/assets/ee8a5fe2-6228-4c84-b5ee-34fcab62fd15" />